guides/Tong/README.md

```diff
@@ -0,0 +1,76 @@
+## Written by Sharon Tong
+_________________________
+
+### Contact
+- Slack: @Sharon on wcscusf.slack.com
+- LinkedIn: linkedin.com/in/sharon-tong
+- Email: sharont1@mail.usf.edu
+### Table of Contents
+1. [Prerequisites](#id-link-to-section)
```

Comment:
Seems I made the mistake to have that in my template. `id` in `<a id="something">` means it's a key link to another item so when they click on it it takes them to that other item. example:
```
1. [Prerequisites](#prereq)
2. [Summary](#summary)

## Prerequisites <a id="prereq"></a>
## Summary <a id="summary"></a>
```